.ci/travis/travis.sh

```diff
@@ -560,6 +560,16 @@ git-status)
   fi
   ;;
 
+validate-ci-temp-empty)
```

Comment:
Nothing is calling this.

Reply:
Midterms are taking their toll; my apologies.  I've added the job to travis.yml.